lib/ansible/modules/network/fortimanager/fmgr_wanopt.py

```diff
@@ -0,0 +1,836 @@
+      - choice | fix | Select fixed data chunking.
+    required: false
+    choices: ["dynamic", "fix"]
+
+  cifs_secure_tunnel:
+    description:
+      - Enable/disable securing the WAN Opt tunnel using SSL. Secure and non-secure tunnels use the same TCP port (781
+      - 0).
+      - choice | disable | Disable SSL-secured tunnelling.
+      - choice | enable | Enable SSL-secured tunnelling.
+    required: false
+    choices: ["disable", "enable"]
+
+  cifs_status:
+    description:
+      - Enable/disable HTTP WAN Optimization.
+      - choice | disable | Disable HTTP WAN Optimization.
+      - choice | enable | Enable HTTP WAN Optimization.
+    required: false
+    choices: ["disable", "enable"]
+
+  cifs_tunnel_sharing:
+    description:
+      - Tunnel sharing mode for aggressive/non-aggressive and/or interactive/non-interactive protocols.
+      - choice | private | For profiles that accept aggressive protocols such as HTTP and FTP so that these aggressiv
+      - choice | shared | For profiles that accept nonaggressive and non-interactive protocols.
+      - choice | express-shared | For profiles that accept interactive protocols such as Telnet.
+    required: false
+    choices: ["private", "shared", "express-shared"]
+
+  ftp:
+    description:
+      - EXPERTS ONLY! KNOWLEDGE OF FMGR JSON API IS REQUIRED!
+      - List of multiple child objects to be added. Expects a list of dictionaries.
+      - Dictionaries must use FortiManager API parameters, not the ansible ones listed below.
+      - If submitted, all other prefixed sub-parameters ARE IGNORED.
+      - This object is MUTUALLY EXCLUSIVE with its options.
+      - We expect that you know what you are doing with these list parameters, and are leveraging the JSON API Guide.
+      - WHEN IN DOUBT, USE THE SUB OPTIONS BELOW INSTEAD TO CREATE OBJECTS WITH MULTIPLE TASKS
+    required: false
+
+  ftp_byte_caching:
+    description:
+      - Enable/disable byte-caching for HTTP. Byte caching reduces the amount of traffic by caching file data sent acr
+      - oss the WAN and in future serving if from the cache.
+      - choice | disable | Disable HTTP byte-caching.
+      - choice | enable | Enable HTTP byte-caching.
+    required: false
+    choices: ["disable", "enable"]
+
+  ftp_log_traffic:
+    description:
+      - Enable/disable logging.
+      - choice | disable | Disable logging.
+      - choice | enable | Enable logging.
+    required: false
+    choices: ["disable", "enable"]
+
+  ftp_port:
+    description:
+      - Single port number or port number range for FTP. Only packets with a destination port number that matches this
+      -  port number or range are accepted by this profile.
+    required: false
+
+  ftp_prefer_chunking:
+    description:
+      - Select dynamic or fixed-size data chunking for HTTP WAN Optimization.
+      - choice | dynamic | Select dynamic data chunking to help to detect persistent data chunks in a changed file or
```

Comment:
The `or` here seems to be redundant since the other choice is mentioned in the next line, can you please take a look at it once more?